cluster/examples/kubernetes/ceph/rgw-external.yaml

```diff
@@ -10,9 +10,9 @@ metadata:
 spec:
   ports:
   - name: rgw
-    port: 80
+    port: 8080 # internal port used by the rgw daemon
```

Comment:
```suggestion
    port: 80 # service port mentioned in object store crd
```

Reply:
@leseb  Sorry I didn't understand why `internal port used by rgw daemon` need to be removed here. IMHO it better to mention both details in `rgw-external.yaml`

Reply:
It needs to be removed because it is not correct, I told you it was the other way around so your comment is not wrong it's just reversed.